gravitee-apim-repository/gravitee-apim-repository-mongodb/src/main/java/io/gravitee/repository/mongodb/management/internal/eventLatest/event/EventLatestMongoRepositoryImpl.java

```diff
@@ -82,40 +84,4 @@ public List<EventLatestMongo> search(EventCriteria criteria, Event.EventProperti
 
         return events.getMappedResults();
     }
-
-    protected List<Criteria> buildDBCriteria(final EventCriteria criteria) {
```

Comment:
The range criteria for Event is different from that for LatestEvent, where LatestEvent uses 'less than or equal to' and Event uses 'less than'. This leads to an almost duplication. I suggest reverting the criteria to avoid potential side effects.

Reply:
Let's discuss it further because I thought it was a kind of bug to have different rules to search events and search the latest events when specifying dates. Could you remind me of the reasons for such a difference?